### PR TITLE
Adds support for TMP006

### PR DESCRIPTION
**Pull request by caternuson**
Notable deviations from TMP007:
* added `active` property and privatized `wake()` and `sleep()`
* add a wait for conversion complete check, currently with no timeout (scary)
* throw an exception if a read is attempted while sensor is asleep

### CODE REVIEW

File: adafruit_tmp006.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    def temperature(self):
+        # pylint: disable=bad-whitespace, invalid-name, too-many-locals
+        """Read object temperature from TMP006 sensor."""
+        if not self.active:
+            raise RuntimeError('Can not read from sensor when inactive.')
+        while not self._data_ready():
+            pass
+        vobj = self.read_register(_TMP006_VOBJ)
+        vobj = bytearray((vobj & 0xFF, vobj >> 8 & 0xFF))
+        vobj = struct.unpack("<h", vobj)[0]
```
Comment (ladyada) [inline adafruit_tmp006.py]: why bytearray swap and then unpack? can just pick one or the other? you can do endian swaps in 'unpack'

File: adafruit_tmp006.py [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        vobj = vobj * 156.25e-9  # volts
+        tamb = self.read_register(_TMP006_TAMB)
```
Comment (ladyada) [inline adafruit_tmp006.py]: split this into its own function like we do here
https://github.com/adafruit/Adafruit_Python_TMP/blob/master/Adafruit_TMP/TMP006.py#L135

File: adafruit_tmp006.py [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+        VOS = b0 + b1*(tamb - TREF) + b2*(tamb - TREF)**2
+        fVOBJ = (vobj - VOS) + c2*(vobj - VOS)**2
+
+        TOBJ = (tamb**4 + (fVOBJ/S))**0.25
+
+        TOBJ -= 273.15 # back to celsius
+
+        return TOBJ
+
+    def _sleep(self):
```
Comment (ladyada) [inline adafruit_tmp006.py]: _sleep and _wake are basically the same function, combine the code and put into the active.setter?

**Review by ladyada — CHANGES_REQUESTED** [review]

File: adafruit_tmp006.py (hunk 2 above)
Comment (caternuson) [reply adafruit_tmp006.py]: done

File: adafruit_tmp006.py (hunk 3 above)
Comment (caternuson) [reply adafruit_tmp006.py]: good call. i was just lazily reusing the tmp007 funcs.

File: adafruit_tmp006.py (hunk 1 above)
Comment (caternuson) [reply adafruit_tmp006.py]: just using `unpack` for 2s comp. endian swap wasn't intentional. took a slightly different approach in latest refactor.

**Comment by ladyada** [comment]
code looks good to me, did you you try with a sensor?

**Comment by caternuson** [comment]
thanks. yep. tested with a basic little setup.

tmp006_test

**Comment by ladyada** [comment]
sounds good to me, then!

**Review by ladyada — APPROVED** [review]

**Comment by ladyada** [comment]
@kattni wanna pypi/release/bundle?

**Comment by kattni** [comment]
Will do!

**Pull request merged**